### ISSUE TITLE
Recent `regex` update might have break all related ui tests

### ISSUE THREAD

**Issue by J-ZhengLi** [rust-lang/rust-clippy#11118]
### Description

`regex` crate just recieved a `1.9.0` update, however, all test cases under regex.rs does not produce output for some reason.

I tried running `TESTNAME='regex' cargo uitest` on stable branch, and it fails on either windows or linux platform.
Then I manually modify Cargo.lock and set the version of `regex` to 1.8.4, and `regex-syntax` to 0.7.2, and everything works normal.
Which is kinda weird, can someone confirm this please? Should `Cargo.lock` be included in the future?

**Comment by y21** [comment]
yep, apparently the path to `Regex` changed and that broke the lint. It was causing CI to fail for all PRs too, however #11111 just got merged and that should have fixed this. Are you on the latest commit?

**Comment by J-ZhengLi** [comment]
> yep, apparently the path to `Regex` changed and that broke the lint. It was causing CI to fail for all PRs too, however #11111 just got merged and that should have fixed this. Are you on the latest commit? (edit: oh I missed that you said "stable")

ahhh~ glad to know it has already been fixed, but I guess I have to manually set regex version each time until that commit made it to the next stable 🤦‍♂️, thank you though, closing this right now.
